packages/gestalt/src/PageHeaderSubcomponents.js

```diff
@@ -0,0 +1,200 @@
+          {title}
+        </Heading>
+      </Box>
+      <Box
+        display="none"
+        smDisplay="block"
+        dangerouslySetInlineStyle={{ __style: { marginTop: '-7px' } }}
+      >
+        <Heading size="500" lineClamp={1} accessibilityLevel={1}>
+          {title}
+        </Heading>
+      </Box>
+    </Fragment>
+  );
+}
+
+export function PageHeaderThubnail({ thumbnail }: {| thumbnail?: Element<typeof Image> |}): Node {
+  return (
+    <Box display="none" smDisplay="block">
+      <Mask height={48} rounding={2} width={48}>
+        {thumbnail}
+      </Mask>
+    </Box>
+  );
+}
+
+export function PageHeaderBadge({
+  badgeText,
+  badgeTooltipText,
+}: {|
+  badgeText: string,
+  badgeTooltipText?: string,
+|}): Node {
+  return (
+    <Fragment>
+      {badgeText && !badgeTooltipText ? (
+        <Badge text={badgeText} type="info" position="top" />
+      ) : null}
+      {badgeText && badgeTooltipText ? (
+        <Tooltip accessibilityLabel="" text={badgeTooltipText} idealDirection="up">
+          <Badge text={badgeText} type="info" position="top" />
+        </Tooltip>
+      ) : null}
+    </Fragment>
+  );
+}
+
+export function PageHeaderHelperIconButton({
+  helperIconButton,
+}: {|
+  helperIconButton?: {|
+    accessibilityLabel: string,
+    accessibilityControls: string,
+    accessibilityExpanded: boolean,
+    onClick: ({|
+      event:
+        | SyntheticMouseEvent<HTMLButtonElement>
+        | SyntheticKeyboardEvent<HTMLButtonElement>
+        | SyntheticMouseEvent<HTMLAnchorElement>
+        | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+|}): Node {
+  return helperIconButton ? (
+    <IconButton
+      accessibilityControls={helperIconButton.accessibilityControls}
+      accessibilityExpanded={helperIconButton.accessibilityExpanded}
+      accessibilityHaspopup
+      accessibilityLabel={helperIconButton.accessibilityLabel}
+      bgColor="lightGray"
+      icon="question-mark"
+      iconColor="darkGray"
+      onClick={helperIconButton.onClick}
+      size="xs"
+    />
+  ) : null;
+}
+
+export function PageHeaderSubtext({
+  subtext,
+  helperLink,
+}: {|
+  subtext?: string,
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+|}): Node {
+  return (
+    <Box wrap display="none" smDisplay="block">
+      <Text inline>
+        {subtext}{' '}
+        {helperLink ? (
+          <Link
+            inline
+            accessibilityLabel={helperLink.accessibilityLabel}
+            href={helperLink.href}
+            onClick={helperLink.onClick}
+            target="blank"
+          >
+            <span style={{ textDecoration: 'underline' }}>{helperLink.text}</span>
```

Comment:
`<Text underline>`?

Reply:
Underline breaks with inline. 'Cos we changed to Flex, then we can use <Text underline>

Reply:
DONE

Reply:
Just remembered why I didn't use Flex
<img width="300" alt="Screen Shot 2022-03-30 at 8 03 37 PM" src="https://user-images.githubusercontent.com/10593890/160950338-6c21e2e1-b347-44c0-9746-951221ca9cee.png">

Reply:
Are you referring to "Learn more" wrapping? You should be able to fix that by wrapping in `<Flex.Item flex="none">`